README.md

```diff
@@ -43,8 +43,10 @@ To start a new worker pool, you can either use `wpool:start_pool` (if you want t
 * **worker_type**: The type of the worker. The available values are `gen_server`. The default value is `gen_server`. Eventually we'll add `gen_statem` as well.
 * **worker**: The [`gen_server`](http://erldocs.com/current/stdlib/gen_server.html) module that each worker will run and the `InitArgs` to use on the corresponding `start_link` call used to initiate it. The default value for this setting is `{wpool_worker, undefined}`. That means that if you don't provide a worker implementation, the pool will be generated with this default one. [`wpool_worker`](https://hexdocs.pm/worker_pool/wpool_worker.html) is a module that implements a very simple RPC-like interface.
 * **worker_opt**: Options that will be passed to each `gen_server` worker. This are the same as described at `gen_server` documentation.
+* **worker_shutdown**: The `shutdown` option to be used in the child specs of the workers. Defaults to `5000`.
 * **strategy**: Not the worker selection strategy (discussed below) but the supervisor flags to be used in the supervisor over the individual workers (`wpool_process_sup`). Defaults to `{one_for_one, 5, 60}`
 * **pool_sup_intensity** and **pool_sup_period**: The intensity and period for the supervisor that manages the worker pool system (`wpool_pool`). The strategy of this supervisor must be `one_for_all` but the intensity and period may be changed from their defaults of `5` and `60`.
+* **pool_sup_shutdown**: The `shutdown` option to be used for the supervisor that manages the worker pool system (`wpool_pool`). Defaults to `brutal_kill`.
```

Comment:
I see that this description could be clearer. Maybe something like this:

"The `shutdown` option to be used for the supervisor over the individual workers (`wpool_process_sup`). That is, the value set in the child spec for this supervisor, which is specified in its parent supervisor (`wpool_pool`)."

?

Reply:
Yeah… I like that description 👌

Reply:
Great, I'll fix it. Should I ammend or add a new commit?

Reply:
I did a new commit, but feel free to squash on merge @elbrujohalcon .